backend/core/src/listings/listings.service.ts

```diff
@@ -72,22 +114,29 @@ export class ListingsService {
       })
     }
 
-    if (jsonpath) {
-      listings = jp.query(listings, jsonpath)
+    // TODO(https://github.com/CityOfDetroit/bloom/issues/135): Decide whether to remove jsonpath
+    if (params.jsonpath) {
+      listings = jp.query(listings, params.jsonpath)
+    }
+
+    const paginatedListings = {
```

Comment:
Mapping to a DTO is usually done at the controller level and services operate on DB models.

Reply:
Done.